lib/api/models/mediator.rb

```diff
@@ -3,6 +3,11 @@
 module API
   module Models
     class Mediator < ActiveRecord::Base
+      before_create :set_urn_prefix
+
+      def set_urn_prefix
```

Comment:
Given the URN `1234T` I'd argue for the following names:

- 1234
  - I wouldn't call this prefix
  - Maybe URN or ID is more accurate
- T
  - maybe better name is suffix

With that in mind, I don't see the reason to store another column. Isn't it just 2 views on the same data?

If the two parts do need to be stored separately, then I'd go for a column for URN and column for suffix. If it's important for the API to return the URN+SUFFIX, then do that there. This would lead me to think why is this data not already split into two separate columns in the spreadsheet.